eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -156,11 +156,9 @@ stages:
                           artifactName: ${{artifact.name}}-symbols
 
                   steps:
-                    - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
-                      parameters:
-                        SkipCheckoutNone: true
+                    - checkout: self
+                      fetchDepth: 1
```

Comment:
Why this change?

Reply:
reverted to sparse checkout with explicit path